spaceros/Dockerfile

```diff
@@ -132,7 +134,7 @@ ENV PATH "$PATH:/root/src/ikos/install/bin"
 ENV IKOS_SCAN_NOTIFIER_FILES "yes"
 
 # The above installation doesn't properly install the IKOS egg file, so do it manually
-RUN cd /root/src/ikos/install/lib/python3.8/site-packages && python3 /usr/lib/python3/dist-packages/easy_install.py ./ikos-3.0-py3.8.egg
+RUN cd /root/src/ikos/install/lib/python3.8/site-packages && zip ikos-3.0-py3.8.egg ikos-3.0-py3.8.egg-info/* && python3 /usr/lib/python3/dist-packages/easy_install.py ./ikos-3.0-py3.8.egg
```

Comment:
Hmm. Is there an upstream ticket for this? If possible we should help upstream update their installation procedures to be correct. But that doesn't mean we should omit fixing it here while that is pending (and link to the upstream issue/patch).

Reply:
Submitted an issue here: https://github.com/NASA-SW-VnV/ikos/issues/185. 

Also created an issue in the Space ROS repo to track: https://github.com/space-ros/space-ros/issues/29.

Reply:
Thanks very much for opening the issue. I took the liberty of pushing https://github.com/space-ros/docker-images/pull/3/commits/4397c2020262abf257d17b2484ff1907ad4d5925 which updates the egg file comment to include a link to that issue.

Reply:
By the way, I pushed a commit rather than using the suggestion API since the line that I wanted to change wasn't showing me the :heavy_plus_sign: button to add a line comment. If you'd prefer not to have that reference then we can pull the commit off.